extensions/wikia/FacebookClient/scripts/FacebookClient.facebookTags.js

```diff
@@ -12,7 +12,9 @@ mw.hook('wikipage.content').add(function ($content) {
 
 	// Checks for XFBML tags or Facebook Page Plugin tags
 	function facebookTagsOnPage() {
-		var numOfFacebookTags = $content.find('[data-type="xfbml-tag"], .fb-page').length;
+        // All fb social plugins have a class which starts with 'fb-'
+        // eg, fb-like, fb-share-button, etc
+		var numOfFacebookTags = $content.find('[data-type="xfbml-tag"], [class^=fb-]').length;
```

Comment:
True, it's encouraged by the documentation https://api.jquery.com/attribute-starts-with-selector/